### ISSUE TITLE
Infobox field names of all Hindi pages are in English

### ISSUE THREAD

**Issue by tanvibhakta** [spencermountain/wtf_wikipedia#261]
The field names of all Hindi pages that have infoboxes are being returned in English. 

![Screen Shot 2019-03-25 at 1 19 24 PM](https://user-images.githubusercontent.com/13763558/54902965-b9f5d180-4f00-11e9-97bc-66ecead6a376.png)

This is a link to the official page:
https://hi.wikipedia.org/wiki/%E0%A4%AE%E0%A4%82%E0%A4%97%E0%A4%B2%E0%A4%AF%E0%A4%BE%E0%A4%A8

![Screen Shot 2019-03-25 at 1 19 38 PM](https://user-images.githubusercontent.com/13763558/54902971-bcf0c200-4f00-11e9-8d72-ec86fc060b56.png)
This is a link to the official page:
https://hi.wikipedia.org/wiki/%E0%A4%AE%E0%A4%B9%E0%A4%BE%E0%A4%A4%E0%A5%8D%E0%A4%AE%E0%A4%BE_%E0%A4%97%E0%A4%BE%E0%A4%82%E0%A4%A7%E0%A5%80

**Comment by spencermountain** [comment]
ah, yeah the templates are keyed in english - which seems like a gross oversight by wikipedia.

They must have a mapping for each language somewhere in the parsoid project, or maybe elsewhere, but I haven't been able to find it.

here's the wikiscript, for reference - we're simply grabbing whatever it has right now.
![image](https://user-images.githubusercontent.com/399657/54998825-6a1d1600-4fa5-11e9-94f0-161930e8a2df.png)

Even in english, the presentation of infoboxes are rendered in ad-hoc ways. I think it probably should be handled in a secondary library - like `wtf-infobox-render` or something, as I imagine it's quite complicated.

[related](https://github.com/spencermountain/wtf_wikipedia/issues/173)

**Comment by spencermountain** [comment]
i emailed one of the parsoid people about this
cheers

**Comment by spencermountain** [comment]
yeah, it's not as simple as I had hoped, the templates get copy+pasted, and the translations are holed-up [like this](https://hi.wikipedia.org/w/index.php?title=%E0%A4%B8%E0%A4%BE%E0%A4%81%E0%A4%9A%E0%A4%BE:%E0%A4%9C%E0%A5%8D%E0%A4%9E%E0%A4%BE%E0%A4%A8%E0%A4%B8%E0%A4%82%E0%A4%A6%E0%A5%82%E0%A4%95_%E0%A4%85%E0%A4%A8%E0%A5%8D%E0%A4%A4%E0%A4%B0%E0%A4%BF%E0%A4%95%E0%A5%8D%E0%A4%B7_%E0%A4%89%E0%A4%A1%E0%A4%BC%E0%A4%BE%E0%A4%A8&action=edit):
```
 |data7     = {{{mission_duration<includeonly>|</includeonly>}}}
 |label8    = मापी गयी दूरी
```
I don't see an easy multilingual fix. Open to ideas
